stdlib/src/builtin/int_literal.mojo

```diff
@@ -269,6 +269,47 @@ struct IntLiteral(
         """
         return self
 
+    @always_inline("nodebug")
```

Comment:
Can you please add one or two simple tests for this? If you want, you can split it out into a separate patch, to get it in faster, since it's a simple change.

Reply:
Sure: https://github.com/msaelices/mojo/commit/78e4ee596860fa305c6548871f05cbdfcc484ca0